src/analyses/interval_domain.cpp

```diff
@@ -78,6 +78,7 @@ void interval_domaint::transform(
 
   case GOTO:
     {
+      // GOTO implies from and to are comparable
       locationt next=from;
       next++;
       if(next==to)
```

Comment:
It may be worth rewriting those three lines of code to `if(std::next(from) == to)`